.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-## Pull Request Guidelines
+<!--
+Checklist for submitting a Pull Request
+
+Please remove this comment block before submitting.
 
 1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
-1. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
-1. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
+3. If you want this PR to automatically close an issue when it is merged,
+   add `fixes #<issue number>` or `fixes #<issue number>, fixes #<issue number>`
+   to close multiple issues (see: https://github.com/blog/1506-closing-issues-via-pull-requests).
+4. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below.
+-->
```

Comment:
Hi @david-mcmahon , yes I comment out the whole list of instructions because you don't want any of them in your final PR. I am trying to show the developer (a bit like git commit) that what is in a comment is "for you info only" and what you need to leave is just the release-note.